app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
   after_save :update_post_counter
+  validates :title, presence: true, length: { maximum: 250 }
+
```

Comment:
### [REQUIREMENT]
Good job for adding validation for the title, likescounter, and commentcounter. However, there's one validation is missing which is about "Title must not exceed 250 characters." as it's part of project's requirements.

<img width="300" alt="Screenshot 2023-05-13 at 5 14 03 AM" src="https://github.com/alicemirigo92/blog_post/assets/83048208/fad4ab4e-afa2-4563-af8e-7bf58298c059">

Reply:
@zainsadaqat Hey this is implemented